Problem 176: Explain practical use of blending direction.

kegor opened the pull request:
Possible fix for #176.
In the added text were used sentences taken from https://github.com/Starcounter/RebelsLounge/issues/153.

Commit:
Problem 176: practical use of blending direction should be explained.

Solution: section about blending direction in docs was updated with the example of practical use.

warpech commented on docs/topic-guides/blendable-web-apps/view-model-attaching.md:
```diff
@@ -170,6 +170,12 @@ Blender.MapUri("/noparam1", token3, true, false);
 
+### Practical use
+
+If you have an app which specifies uni-directional mapping `Blender.MapUri(..., "settings", false ,true)`, then a Self.GET to your own URL will not return other views.
```
Few corrections:

If you have an app which specifies a uni-directional rule `Blender.MapUri(..., "settings", false, true)`, then a `Self.GET` call to your own URL will not return other view-models.

warpech commented on docs/topic-guides/blendable-web-apps/view-model-attaching.md:
```diff
@@ -170,6 +170,12 @@ Blender.MapUri("/noparam1", token3, true, false);
+
+If you have an app which specifies uni-directional mapping `Blender.MapUri(..., "settings", false ,true)`, then a Self.GET to your own URL will not return other views.
+
+Furthermore let's imagine that in the same database, you have another running app (let's call it **Settings**) which should show settings from all other running apps, but do not has self settings for example, then it should has a uni-directional mapping in the opposite direction `Blender.MapUri(..., "settings", true, false))`. In this case, it will pull all settings views from all apps, but won't ...
```
Few grammar corrections and changed the sample app to be "Launchpad" (actual case):

Furthermore, let's imagine that in the same database, you have an app (Launchpad) which shows settings pages from other apps, but does not provide a settings page of its own. It should have a uni-directional rule in the opposite direction: `Blender.MapUri(..., "settings", true, false))`. In this case, it pulls settings view-models from other apps, but won't respond when the `settings` token is called from another app.

Commit:
Problem 176: few sentences should be corrected.

Solution: sample "Settings" app was changed to "Launchpad", occurrences of the "views" were replaced by "view-models".

kegor commented:
Thank you for the corrections @warpech, changes were made.

warpech reviewed (approved):
Lgtm
